Optional display of pylint code, and human-readable version.

bmhkim opened the pull request:
Added display of pylint code, and human-readable (short) code, depending on new setting options. This solves issue #28.

posita commented on lib/init.coffee:
```diff
@@ -6,6 +6,14 @@ module.exports =
       type: 'string'
       default: ''
+    showCodes:
+      description: "Show pylint codes? (E1101, etc.)"
```
Consider instead something like:

title: "Show Pylint Codes"
description: "Displays the Pylint code in the Linter message  (e.g., E1101, etc.)"

posita commented on lib/init.coffee:
```diff
@@ -6,6 +6,14 @@ module.exports =
+      type: 'boolean'
+      default: false
+    showReadableCodes:
+      description: "Show the short human-readable version of pylint codes? (e.g., unused-import for W0611)"
```
Consider instead something like:

title: "Show Pylint Symbols"
description: "Displays the human-readable Pylint symbol in the Linter message  (e.g., unused-import, etc.)"

posita commented:
Not my rodeo, but this gets a :+1: from me. Consider my above suggestions regarding the separating the title and descriptions for the settings. Also consider [squashing your second (and any subsequent) commit](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html) to make this PR more attractive to the maintainer.

bmhkim commented:
Thanks for the comments, I appreciate the feedback. Since I agree with all you've said, I've updated the option descriptions to be declarative statements describing the behavior being controlled, and squashed my commits. I imagine you'll need to ignore some git warnings....

posita commented:
LGTM

@dmnd, thoughts?

SpainTrain commented:
Should this be closed now that #47 is merged?

posita commented:
@SpainTrain, is it validated as working and released?

SpainTrain commented:
Definitely not released.

I tried to validate it works, but I think an unrelated change that's currently on master made the plugin not work at all (at least in my environment).

Agree it makes sense to keep it open until some folks can verify that #47 actually works.

SpainTrain commented:
> @SpainTrain, is it validated as working and released?

Approach in #47 works for me.  Same approach works in #55. One validation data point X-D

SpainTrain commented:
Definitely appreciate the work, but the code has changed significantly since this was opened and similar functionality was implemented. Closing.
